### PR TITLE
(FM-7755) - Port Channel update to account for XE

### PR DESCRIPTION
**Pull request by david22swan**
- Exclusions added for flow control send
- Test update

### CODE REVIEW

File: spec/acceptance/port_channel_spec.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@
     expect(result).to match(%r{flowcontrol_receive.*on}) if result =~ %r{flowcontrol_receive}
     expect(result).to match(%r{flowcontrol_send.*on}) if result =~ %r{flowcontrol_send}
     expect(result).to match(%r{speed.*10}) if result =~ %r{speed}
-    expect(result).to match(%r{mode.*passive}) if result =~ %r{mode}
+    expect(result).to match(%r{mode.*passive}) if result =~ %r{mode =>}
```
Comment (willmeek) [inline spec/acceptance/port_channel_spec.rb]: Good catch that we are now matching the entire debug output.

Could we change speed and the 2 flowcontrol checks too to include the arrow?

Thanks!

Comment (david22swan) [reply spec/acceptance/port_channel_spec.rb]: Updated all flow controls to include the `=>`

File: README.md [hunk 2]
```diff
@@ -1113,10 +1113,20 @@ Does not support the following attributes: [link](https://www.cisco.com/c/en/us/
...
+
 ##### 3750
 
+##### 4503
+
 ##### 4507
 
+##### 4948
+
 This device does not have native trunking. It does not support the following attributes: [link](https://learningnetwork.cisco.com/thread/75947)
```
Comment (da-ar) [inline README.md]: This line looks like it belongs to the 4507 in the original readme? Should it be moved up?

Comment (david22swan) [reply README.md]: It's shared by all of them. Thought this was better than repeating it 10 times.

Comment (da-ar) [reply README.md]: OK, then reword this to: 
"The above devices do not have native trunking. The following attributes are not supported : [link](https://learningnetwork.cisco.com/thread/75947)"

Comment (david22swan) [reply README.md]: Updated
